app/models/site.rb

```diff
@@ -11,11 +11,27 @@ class Site < ActiveRecord::Base
            :announcement_text=, :marketing_text=, :featured_researcher=,
            to: :content_blocks
 
+  def content_by_name(name)
```

Comment:
@dlpierce Can you write a bit about what the new methods add? We didn't need these for the first few content blocks, is why I ask. Is this a potential or partial refactoring?